README.md

```diff
@@ -1,34 +1,30 @@
 # RHTAP Observability 
 
-This repository contains Prometheus recording rule files and alert rule files for
-observability use, as well as unit tests for these rules.
+This repository is maintained by the RHTAP O11Y team.
 
-## Recording Rules
+The repository contains Prometheus alert rules [files](rhobs/alerting) for monitoring
+RHTAP data plane clusters along with their [tests](test/promql).
 
-- `appstudio_container_network_transmit_bytes_total` collects the network egress of
-containers and adds the `label_pipelines_appstudio_openshift_io_type` label to our
-metric.
+Control plane clusters alert rules are maintained as well by the same team, but are
+kept in a different
+[repository](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/resources/stonesoup/argocd-control-plane/monitoring)
 
-- `appstudio_container_resource_limits` collects all the resource limits
-(cpu and memory) for all containers and init_containers, and adds
-the `label_pipelines_appstudio_openshift_io_type` label to the new metric
+## Updating Data Plane Alerts
 
-- `appstudio_container_resource_minutes_gauge` uses the resource limit configured on
-the container, as captured by `appstudio_container_resource_limits`, and transforms it to a
-fixed time window gauge representing the resource-limit per minute multiplied by the
-period the container was alive in that time window.
+Alert rules for data plane clusters are being deployed by app-interface to RHOBS, to where the data plane metrics are also being forwarded. For deploying the alert rules,
+app-interface references the location of the rules together with a git reference -
+branch name or commit hash.
 
-Examples (assuming 1-minute time window)
-- A container lived for the entire minute and had a limit of 0.5 CPU cores --> 0.5 CPU
-minutes.
-- A container lived for 30 seconds within the measured time frame and had a limit of
-2 CPU cores --> 1 CPU minutes.
-This metric is intended to then be summed over the time frame the metric consumer wants
-to measure for.
+It holds separate references to both staging and production RHOBS instances (monitoring
+RHTAP staging and production deployments). For both cases, we maintain the reference to
```

Comment:
Current: "For both cases"

Suggestion: "In both cases"

Reply:
I don't think that fits here. I changed to something else.